### PR TITLE
Fix spec link in showOpenFilePicker doc

### LINKED ISSUE #2337: Issue with "Window.showOpenFilePicker()": spec link appears to be broken
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Window/showOpenFilePicker

#### What information was incorrect, unhelpful, or incomplete?
AFAICT, the spec should point [here](https://wicg.github.io/file-system-access/#api-showopenfilepicker). Currently, it appears to point to an invalid page with the title "Unknown"

#### Specific section or headline?
Specifications

#### What did you expect to see?
Link to some page, with appropriate title.

#### Did you test this? If so, how?
No

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/window/showopenfilepicker`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/Window/showOpenFilePicker
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/api/window/showopenfilepicker/index.html
* Last commit: https://github.com/mdn/content/commit/2d423bd1a98b85b8584f08990792ebf39fcea56b
* Document last modified: 2021-01-09T13:41:23.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/2337

### CODE REVIEW

File: files/en-us/web/api/window/showopenfilepicker/index.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
-- Method
-- Window
-- working with files
+  - File
+  - File System Access API
+  - Method
+  - Window
+  - working with files
 ---
 <div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("File System Access API")}}
```
Comment (wbamberg) [inline files/en-us/web/api/window/showopenfilepicker/index.html:11]:
```suggestion
<div>{{securecontext_header}}{{DefaultAPISidebar("File System Access API")}}
```

Maybe we can remove Draft here? This page looks reasonably complete.

**Review by wbamberg — APPROVED** [review]
Looks great, thank you @sideshowbarker . I suggested maybe we could remove the Draft banner.

File: files/en-us/web/api/window/showopenfilepicker/index.html (hunk 1 above)
Comment (sideshowbarker) [reply files/en-us/web/api/window/showopenfilepicker/index.html]: > Maybe we can remove Draft here? This page looks reasonably complete.

Yup — so removed

**Comment by wbamberg** [comment]
Thanks!

**Pull request merged**